Finjector.Web/ClientApp/src/components/Shared/ChartList.tsx

```diff
@@ -60,7 +60,7 @@ const ChartList = (props: Props) => {
   return (
     <div>
       {filteredTeamGroups.map((teamGroup) => (
-        <div className="teamlist-wrapper">
+        <div className="teamlist-wrapper" key={teamGroup.team.id}>
```

Comment:
@laholstege I think this is wrong as it may conflict with key={chart.id} below?
Prefix it with something so it is unique.

Reply:
keys only need to be unique among their siblings (learned in my course). react is using it to keep track of the items as they move/update–using the index of the array is not recommended because the index won't correspond to the actual item, just the position. but the keys only need to be unique within the array we're doing .map() on, so teamId will uniquely identify each team at this level, and chartId will uniquely identify for the lower .map()–they don't need to be different 
https://react.dev/learn/rendering-lists#rules-of-keys